class.jetpack-twitter-cards.php

```diff
@@ -196,9 +198,14 @@ static function sharing_global_options() {
 	static function site_tag() {
 		$site_tag = get_option( 'jetpack-twitter-cards-site-tag' );
 		if ( empty( $site_tag ) ) {
-			$site_tag = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? 'wordpressdotcom' : 'jetpack';
+			if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+				return 'wordpressdotcom';
+			} else {
+				return;
+			}
+		} else {
```

Comment:
The logic would probably read as cleaner if this wans't an else, and just fell through to the prior `return $site_tag`